src/generated/sdk.ts

```diff
@@ -1683,7 +1779,7 @@ export type MutationRemoveMemberFromSepProposalArgs = {
 };
 
 
-export type MutationAssignProposalArgs = {
+export type MutationAssignProposalToSepArgs = {
```

Comment:
I see sometimes we call it Args, and sometimes Input. Is there a difference or just a convention we need to agree upon

Reply:
Well, arg/args are what they sound like, used for arguments on queries and mutation (like pagination, filtering), the input is mostly for mutation where you want to pass a payload, you can use arg/args where you define what is the input.

This example may make more sense: https://typegraphql.com/docs/resolvers.html#arguments

Reply:
But yes, I think it a convention we should follow.